python/seldon_core/user_model.py

```diff
@@ -278,9 +285,14 @@ def client_transform_output(
     return features
 
 
-def client_custom_metrics(user_model: SeldonComponent) -> List[Dict]:
+def client_custom_metrics(
+    user_model: SeldonComponent, seldon_metrics: SeldonMetrics
+) -> List[Dict]:
     """
-    Get custom metrics
+    Get custom metrics for client and update SeldonMetrics.
+
+    This function will return empty list if INCLUDE_METRICS_IN_CLIENT_RESPONSE environmental
+    variable is NOT set to "true" or "True".
 
     Parameters
     ----------
```

Comment:
Should we add a line for the new `seldon_metrics` parameter?

Reply:
Added.